composer.json

```diff
@@ -18,11 +18,10 @@
         "wiki": "https://github.com/koraktor/steam-condenser/wiki"
     },
     "require": {
-        "php": ">=5.4.0",
-        "monolog/monolog": "~1.10"
+        "php": "^7.2",
+        "monolog/monolog": "^2.0"
     },
     "require-dev": {
-        "phpdocumentor/phpdocumentor": "~2.6.1",
```

Comment:
This is unrelated and should go into a separate PR.

Reply:
My apologies, that was removed oin my fork and I forgot to re-add that change.

Reply:
So, i'm happy to make another PR for phpdoc, but what are we even using it for?

I don't see any phpdocs live anywhere, there's no docs written to the repo, what are we doing with it?

I can't install it via composer because it requires monolog 1.6, but reverting all these changes to satisfy a dev dependency just for the convenience of installing it via composer seems silly.

Thoughts?

Reply:
I think updating to 3.0.0-rc should be fine.

Reply:
I actually did do that, though not via composer, let me see if that's a viable option, thanks.

Reply:
I was able to achieve this, but only by setting the minimum stability to dev and telling it to prefer stable packages.

See here: https://github.com/zack6849/steam-condenser-php/tree/travis-php7

I tested it in my branch to bring all the testing code up to 7.x, I can make it a separate PR, but let's tackle that after we figure out this whole monolog thing, too many fixes in the air right now IMO

Reply:
I‘m totally fine with having these changes in a single „update dependencies“ PR.

Looks good so far.